onnx/backend/test.py

```diff
@@ -85,13 +85,14 @@
 
 
 class BackendTest(object):
-    def __init__(self, backend):
+    def __init__(self, backend, parent_module):
```

Comment:
Make `parent_module` optional (default to `None`) and set the `__module__` only if this is not `None`

Reply:
Good catch.